specification/securityinsights/resource-manager/readme.go.md

```diff
@@ -22,5 +23,14 @@ These settings apply only when `--tag=package-composite-v1 --go` is specified on
 Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
 ```yaml $(tag) == 'package-composite-v1' && $(go)
-output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2017-08-01-preview/$(namespace)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/v1.0/$(namespace)
```

Comment:
should it be a breaking change?

Reply:
Yes, but this is a preview package, therefore it should be fine